packages/migrate/src/__tests__/DbCommand.test.ts

```diff
@@ -53,8 +53,5 @@ it('db seed without --preview-feature flag', async () => {
     DbCommand.new({
       dev: DbSeed.new(),
     }).parse(['dev']),
-  ).rejects.toMatchInlineSnapshot(`
-          This feature is currently in Preview. There may be bugs and it's not recommended to use it in production environments.
-          Please provide the --preview-feature flag to use this command.
-        `)
+  ).rejects.toThrowError()
```

Comment:
Why is this erroring? (I haven't read the whole test, might be missing something obvious)

Reply:
In this situation it throws because no seed is configured, it's annoying to snapshot this here because it's already done in DbSeed test.

Reply:
Should we throw in that case? I would have imagined we just print something to the console and exit. If that's what it does and there is a snapshot test for that case then all fine :+1:

Reply:
I think I saw the test, so conversation resolved :heavy_check_mark: